src/api/common/HttpClient.ts

```diff
@@ -86,13 +88,19 @@ class HttpClient {
   ): Promise<string> {
     const url = buildUrl(relayUrl, PATHS.POST_RELAY_REQUEST);
 
-    const { signedTx } =
+    const { signedTx, error, message } =
       await this._httpWrapper.sendPromise<SignedTransactionDetails>(
         url,
         this._stringifyEnvelopingTx(envelopingTx)
       );
     log.info('relayTransaction response:', signedTx);
 
+    const errMsg = error ?? message;
```

Comment:
Since we're calling the `/relay` handler, the error should be stored in the `error` property, hence `message` should be always undefined. It looks like we do set the `message` property in the `/tokens` handler only, but we're using it from the client at all.

Reply:
removed case for message.